tests/integration/mounts_propagation.bats

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+        requires root
+        setup_debian
+}
+
+function teardown() {
+        teardown_bundle
+}
+
+@test "runc run [rootfsPropagation shared]" {
+        # Set rootfsPropagation to shared
+        update_config ' .linux.rootfsPropagation = "shared" '
+
+        update_config ' .process.args = ["sh", "-c", "findmnt --noheadings -o PROPAGATION /"] '
+
+        # Run the container and capture the output
+        runc run test_shared_rootfs
+        [ "$status" -eq 0 ]
+        [[ "$output" == *"shared"* ]]
```

Comment:
```suggestion
        [ "$output" == "shared" ]
```

Reply:
strictly speaking this should be
```bash
[ "$output" = "shared" ]
```
(i.e. single `=`), since `test` (aka `[`) only support `=` for string comparison (unlike `[[` which supports `==`).